### ISSUE TITLE
Freezes at 'css/normalize.css'

### ISSUE THREAD

**Issue by Tyrael84** [silexlabs/Silex#439]
Just started using Silex today and I REALLY like it as an editor. It's perfect for what I need. But when I finally finished my homepage and wanted to publish it so I could see what it looks like in a browser it freezes at 

'normalize.css'

??? 

UPDATE: Well this whole ordeal really bummed me out (I stayed up till like 3 am trying to figure it out) but I just woke up and tried it again and it worked flawlessly. Must have been my computer or internet connection.

Thanks, singchan, for the fast response!

Thank you Silex for the great website tool I think I'm in love! :)

**Comment by singchan** [comment]
hello
glad you like it :)
do you have the published version online somewhere? can I see it?

**Comment by Tyrael84** [comment]
Unfortunately that's the problem! I've got the first page looking how I like it. I've got my dropbox folder set up (file>settings). But when I go to file>publish it will download everything to my dropbox (assets mostly) and then when it gets to css/normalize.css it just freezes. I thought maybe it was a big file so I let it run and I was able to cook an entire meal (30-45 minutes) and it was still stuck there. :(
